files/file.go

```diff
@@ -271,7 +272,23 @@ func (i *FileInfo) detectSubtitles() {
 	i.Subtitles = []string{}
 	ext := filepath.Ext(i.Path)
 
-	// TODO: detect multiple languages. Base.Lang.vtt
+	// detect multiple languages. Base.Lang.vtt
+	// TODO: give subtitles descriptive names (lang) and track attributes
+	afs := &afero.Afero{Fs: i.Fs}
+	parentDir := strings.TrimRight(i.Path, i.Name)
+	dir, err := afs.ReadDir(parentDir)
+	if err == nil {
+		base := strings.TrimSuffix(i.Name, ext)
+		r := regexp.MustCompile("^" + regexp.QuoteMeta(base) + `\.(.*\.)?vtt$`)
```

Comment:
`MustCompile` must not be used in user-provided data. Use `Compile` instead. I think we can use a simple substring check which would be ~200 times faster than regexp https://go.dev/play/p/HOZBYFaGH1Z

Reply:
I agree. This would eliminate the restriction to `Base.Lang.vtt`.